### PR TITLE
Fixes issue #1255

### PR DESCRIPTION
**Pull request by marianopeck**

### CODE REVIEW

File: repository/BaselineOfSeaside3.package/BaselineOfSeaside3.class/instance/baselinescriptaculous..st [hunk 1]
```diff
@@ -7,7 +7,7 @@ baselinescriptaculous: spec
 			package: 'Prototype-Core' with: [
 				spec requires: #('Javascript-Core' ) ];
 			package: 'Prototype-Tests-Core' with: [
-				spec requires: #('Prototype-Core' 'Javascript-Tests-Core' 'Seaside-Tests-Functional') ];
+				spec requires: #('Prototype-Core' 'Javascript-Tests-Core' 'Seaside-Tests-Functional' 'Seaside-Test-Cavas' ) ];
```
Comment (jbrichau) [inline repository/BaselineOfSeaside3.package/BaselineOfSeaside3.class/instance/baselinescriptaculous..st:10]: That looks like two typos in the package name :-)

**Review by jbrichau — CHANGES_REQUESTED** [review]
All builds fail because the package name was spelled incorrectly.

**Comment by marianopeck** [comment]
hahahahaha I am  the worst @jbrichau ... but...give me the world record guinness :)

Do you want me to submit again or you prefer doing it?

**Comment by jbrichau** [comment]
@marianopeck If you don't have the branch anymore, I can do it, but if you could just push the change to this branch that would be quicker :-)

**Comment by marianopeck** [comment]
Thanks @jbrichau I just pushed the typos fixes. Let me know if it worked.

**Review by jbrichau — APPROVED** [review]

**Pull request merged**